<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="59130" started-at="2020-03-28T14:42:04Z" finished-at="2020-03-28T14:43:03Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="59130" started-at="2020-03-28T14:42:04Z" finished-at="2020-03-28T14:43:03Z">
      <class name="testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester">
        <test-method status="PASS" signature="BeforeSuite()[pri:0, instance:testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester@1d73d1a]" name="BeforeSuite" is-config="true" duration-ms="9433" started-at="2020-03-28T22:41:44Z" finished-at="2020-03-28T22:41:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeSuite -->
        <test-method status="PASS" signature="setCookie()[pri:0, instance:testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester@1d73d1a]" name="setCookie" is-config="true" duration-ms="10501" started-at="2020-03-28T22:41:54Z" finished-at="2020-03-28T22:42:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setCookie -->
        <test-method status="FAIL" signature="ywsl_xcsl_save_ssjbSave_success_test_case(testcase.ywsl.xcsl.save.ssjbSave.YwslXcslSaveSsjbSaveSuccessData)[pri:0, instance:testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester@1d73d1a]" name="ywsl_xcsl_save_ssjbSave_success_test_case" duration-ms="39484" started-at="2020-03-28T22:42:23Z" data-provider="dp" finished-at="2020-03-28T22:43:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[YwslXcslSaveSsjbSaveSuccessData [dwlx=1, yjgsd=北京市, ejgsd=市辖区, sjgsd=东城区, name=五柳先生, phone=13325811144, nsrsbh=332032655632365000, msgNumber=13325811144, czNumber=13325811144, address=东城区前门大街108号, sfzNumber=330521199810142614, workPlace=故宫博物院, email=626262626@qq.com, ywlb=4, nsrpjdj=0, ssjblx=纳税人一般税收违法行为举报, jbejfl=2, bjjrxm=王宝强, bjjrzs=东城区南池子街125号, bjbdw=, ryjb=, zwgw=, bjjjgmc=, jgjb=, sfyzjzl=0, sfyytgzl=1, jbdx=, sfysqxzfy=, sfytqxzss=, sfctcxfhjb=, xhdwtc=, xfhjbjyjg=, jbnr=自动化测试-jbnr, jjcd=紧急, sfcyy=, sffj=, tips=保存成功。]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 2, Size: 2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 2, Size: 2
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at base.BaseTester.switchToHandle(BaseTester.java:357)
	at testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester.ywsl_xcsl_save_ssjbSave_success_test_case(SsjbSaveTester.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ywsl_xcsl_save_ssjbSave_success_test_case -->
        <test-method status="FAIL" signature="ywsl_xcsl_save_ssjbSave_success_test_case(testcase.ywsl.xcsl.save.ssjbSave.YwslXcslSaveSsjbSaveSuccessData)[pri:0, instance:testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester@1d73d1a]" name="ywsl_xcsl_save_ssjbSave_success_test_case" duration-ms="221" started-at="2020-03-28T22:43:03Z" data-provider="dp" finished-at="2020-03-28T22:43:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[YwslXcslSaveSsjbSaveSuccessData [dwlx=1, yjgsd=北京市, ejgsd=市辖区, sjgsd=东城区, name=五柳先生, phone=13325811144, nsrsbh=332032655632365000, msgNumber=13325811144, czNumber=13325811144, address=东城区前门大街108号, sfzNumber=330521199810142614, workPlace=故宫博物院, email=626262626@qq.com, ywlb=4, nsrpjdj=1, ssjblx=轻微纳税人税收违法行为举报, jbejfl=2, bjjrxm=王宝强, bjjrzs=东城区南池子街126号, bjbdw=, ryjb=, zwgw=, bjjjgmc=, jgjb=, sfyzjzl=1, sfyytgzl=0, jbdx=, sfysqxzfy=, sfytqxzss=, sfctcxfhjb=, xhdwtc=, xfhjbjyjg=, jbnr=自动化测试-jbnr, jjcd=一般, sfcyy=1, sffj=, tips=保存成功。]]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[unexpected alert open: {Alert text : }
  (Session info: chrome=79.0.3945.117)
  (Driver info: chromedriver=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}),platform=Windows NT 6.1.7601 SP1 x86) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 7 milliseconds: null
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'PC201703061101', ip: '192.168.8.75', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_181'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:42673}, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}), userDataDir=C:\Users\ADMINI~1\AppData\Local\Temp\scoped_dir15016_463985361}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=79.0.3945.117, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true}]
Session ID: 4b1c7849611c37cb890dbdb86144fab5]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : }
  (Session info: chrome=79.0.3945.117)
  (Driver info: chromedriver=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}),platform=Windows NT 6.1.7601 SP1 x86) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 7 milliseconds: null
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'PC201703061101', ip: '192.168.8.75', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_181'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:42673}, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}), userDataDir=C:\Users\ADMINI~1\AppData\Local\Temp\scoped_dir15016_463985361}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=79.0.3945.117, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true}]
Session ID: 4b1c7849611c37cb890dbdb86144fab5
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:164)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions$RemoteWindow.maximize(RemoteWebDriver.java:945)
	at base.BaseTester.windowsMaximize(BaseTester.java:125)
	at testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester.ywsl_xcsl_save_ssjbSave_success_test_case(SsjbSaveTester.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ywsl_xcsl_save_ssjbSave_success_test_case -->
        <test-method status="FAIL" signature="ywsl_xcsl_save_ssjbSave_success_test_case(testcase.ywsl.xcsl.save.ssjbSave.YwslXcslSaveSsjbSaveSuccessData)[pri:0, instance:testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester@1d73d1a]" name="ywsl_xcsl_save_ssjbSave_success_test_case" duration-ms="46" started-at="2020-03-28T22:43:03Z" data-provider="dp" finished-at="2020-03-28T22:43:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[YwslXcslSaveSsjbSaveSuccessData [dwlx=1, yjgsd=北京市, ejgsd=市辖区, sjgsd=东城区, name=五柳先生, phone=13325811144, nsrsbh=332032655632365000, msgNumber=13325811144, czNumber=13325811144, address=东城区前门大街108号, sfzNumber=330521199810142614, workPlace=故宫博物院, email=626262626@qq.com, ywlb=4, nsrpjdj=2, ssjblx=税务机关和税务人员税收违法行政行为举报, jbejfl=2, bjjrxm=王宝强, bjjrzs=东城区南池子街127号, bjbdw=12366北京中心, ryjb=科级领导干部, zwgw=运维, bjjjgmc=12366北京中心省税务机关, jgjb=, sfyzjzl=0, sfyytgzl=1, jbdx=, sfysqxzfy=, sfytqxzss=, sfctcxfhjb=, xhdwtc=, xfhjbjyjg=, jbnr=自动化测试-jbnr, jjcd=急, sfcyy=, sffj=1, tips=保存成功。]]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[unexpected alert open: {Alert text : }
  (Session info: chrome=79.0.3945.117)
  (Driver info: chromedriver=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}),platform=Windows NT 6.1.7601 SP1 x86) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 14 milliseconds: null
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'PC201703061101', ip: '192.168.8.75', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_181'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:42673}, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}), userDataDir=C:\Users\ADMINI~1\AppData\Local\Temp\scoped_dir15016_463985361}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=79.0.3945.117, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true}]
Session ID: 4b1c7849611c37cb890dbdb86144fab5]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : }
  (Session info: chrome=79.0.3945.117)
  (Driver info: chromedriver=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}),platform=Windows NT 6.1.7601 SP1 x86) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 14 milliseconds: null
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'PC201703061101', ip: '192.168.8.75', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_181'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:42673}, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}), userDataDir=C:\Users\ADMINI~1\AppData\Local\Temp\scoped_dir15016_463985361}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=79.0.3945.117, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true}]
Session ID: 4b1c7849611c37cb890dbdb86144fab5
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:164)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:316)
	at base.BaseTester.to(BaseTester.java:505)
	at testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester.ywsl_xcsl_save_ssjbSave_success_test_case(SsjbSaveTester.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ywsl_xcsl_save_ssjbSave_success_test_case -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester@1d73d1a]" name="afterSuite" is-config="true" duration-ms="7578" started-at="2020-03-28T22:43:04Z" finished-at="2020-03-28T22:43:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testcase.ywsl.xcsl.save.ssjbSave.SsjbSaveTester -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
